### ISSUE TITLE
static rendering adds a leading slash to all Links

### ISSUE THREAD

**Issue by threepointone** [ReactTraining/react-router#4516]
## Version
4.0.0-beta.5

## Test Case

```jsx
renderToStaticMarkup(<StaticRouter location='/' context={{}}>
  <Link to='/a'>to a</Link>
</StaticRouter>)
```
## Steps to reproduce
render a react component containing `<Link/>`s
## Expected Behavior
the string should be `<a href="/a">to a</a>`
## Actual Behavior
the string is `<a href="//a">to a</a>` (note the extra slash)

(the problem exists with `renderToString` as well, used the above for brevity)

**Comment by pshrmn** [comment]
Fixed in #4484

**Comment by threepointone** [comment]
thanks much!
